gcp-observability/src/main/proto/grpc/observabilitylog/v1/observabilitylog.proto

```diff
@@ -28,151 +28,99 @@ option java_outer_classname = "ObservabilityLogProto";
 message GrpcLogRecord {
   // List of event types
   enum EventType {
-    GRPC_CALL_UNKNOWN = 0;
+    EVENT_TYPE_UNKNOWN = 0;
     // Header sent from client to server
-    GRPC_CALL_REQUEST_HEADER = 1;
+    CLIENT_HEADER = 1;
     // Header sent from server to client
-    GRPC_CALL_RESPONSE_HEADER = 2;
+    SERVER_HEADER = 2;
     // Message sent from client to server
-    GRPC_CALL_REQUEST_MESSAGE = 3;
+    CLIENT_MESSAGE = 3;
     // Message sent from server to client
-    GRPC_CALL_RESPONSE_MESSAGE = 4;
-    // Trailer indicates the end of the gRPC call
-    GRPC_CALL_TRAILER = 5;
+    SERVER_MESSAGE = 4;
     // A signal that client is done sending
-    GRPC_CALL_HALF_CLOSE = 6;
+    CLIENT_HALF_CLOSE = 5;
+    // Trailer indicates the end of the gRPC call
+    SERVER_TRAILER = 6;
     // A signal that the rpc is canceled
-    GRPC_CALL_CANCEL = 7;
+    CANCEL = 7;
   }
+
   // The entity that generates the log entry
   enum EventLogger {
     LOGGER_UNKNOWN = 0;
-    LOGGER_CLIENT = 1;
-    LOGGER_SERVER = 2;
-  }
-  // The log severity level of the log entry
-  enum LogLevel {
-    LOG_LEVEL_UNKNOWN = 0;
-    LOG_LEVEL_TRACE = 1;
-    LOG_LEVEL_DEBUG = 2;
-    LOG_LEVEL_INFO = 3;
-    LOG_LEVEL_WARN = 4;
-    LOG_LEVEL_ERROR = 5;
-    LOG_LEVEL_CRITICAL = 6;
+    CLIENT = 1;
+    SERVER = 2;
   }
 
-  // The timestamp of the log event
-  google.protobuf.Timestamp timestamp = 1;
-
-  // Uniquely identifies a call. The value must not be 0 in order to disambiguate
-  // from an unset value.
-  // Each call may have several log entries. They will all have the same rpc_id.
+  // Uniquely identifies a call.
+  // Each call may have several log entries. They will all have the same call_id.
   // Nothing is guaranteed about their value other than they are unique across
   // different RPCs in the same gRPC process.
-  string rpc_id = 2;
+  string call_id = 2;
 
-  EventType event_type = 3;      // one of the above EventType enum
-  EventLogger event_logger = 4;  // one of the above EventLogger enum
+  // The entry sequence ID for this call. The first message has a value of 1,
+  // to disambiguate from an unset value. The purpose of this field is to
+  // detect missing entries in environments where durability or ordering is
+  // not guaranteed.
+  uint64 sequence_id = 3;
 
-  // the name of the service
-  string service_name = 5;
-  // the name of the RPC method
-  string method_name = 6;
+  EventType type = 4;      // one of the above EventType enum
+  EventLogger logger = 5;  // one of the above EventLogger enum
 
-  LogLevel log_level = 7;        // one of the above LogLevel enum
```

Comment:
Looking at the design I didn't see any comments about removing the `log_level`. Was it decided to remove this field?

Reply:
The earlier logging stuff _added_ the field. In the new design (which starts from the binary logging perspective) it was simply not added. It was already vestigial in the previous design.